src/pages/contributing/get-started.mdx

```diff
@@ -0,0 +1,175 @@
+and the process.
+
+## Contribution gallery
+
+In the past year, Carbon contributors have added features to components,
+improved the React testing suite, created image production guidelines in Figma,
+and so much more! Explore the gallery below to see a small sample of finished
+projects.
+
+<iframe
+  class="airtable-embed"
+  src="https://airtable.com/embed/appuREd5slHj4VLfl/shrv9yRexbUjzn4pJ?backgroundColor=cyanDusty&viewControls=on"
+  frameborder="0"
+  onmousewheel=""
+  width="100%"
+  height="533"
+  style="background: transparent; border: 1px solid #ccc;"
+></iframe>
+
+## Carbon champions
+
+Many thanks to our 2023 Carbon champions. Know someone who deserves recognition?
+Fill out the
+[nomination form](https://airtable.com/appOVMXJWJgb6dwhX/shrkMzlvQhPBB6UnX).
+
+- Michael Gower - Accessibility
+- Jess Lin - Accessibility
+- Patrick Clough - Figma
+- Eugene To - Figma
+- Enzo Colasante - Charts
+- Tomas Makan - Charts
+- Colleen Yates - Illustration Guild
+- Matt Gallo - RTL Migration
+- Chris Knabe - RTL Migration
+
+## Types of contribution
+
+There are several ways you can contribute, depending on your interest and
+bandwidth.
+
+### Component enhancements
+
+This work involves enhancing our components in code, like adding new features or
+variants. Designers are often responsible for researching UX and visual
+solutions and then creating a design spec for developers.
+[Here are issues](https://github.com/carbon-design-system/carbon/issues?q=is%3Aissue+is%3Aopen+label%3A%22proposal%3A+accepted%22+label%3A%22community+contribution%22+label%3A%22type%3A+enhancement+%F0%9F%92%A1%22+)
+in our enhancement backlog that need design or development work.
+
+### Design kits
+
+This work is all about creating and updating reusable components is Figma.
+You'll get valuable Figma experience and will collaborate with the Figma Guild.
+Check out the
+[status of design kits](https://www.figma.com/file/CFMtqV5Nztdbm0mi2UiDLg/Library-%2B-Template-Planning?type=design&node-id=3713-26762&mode=design&t=HqvAYXUeccKNMstT-0)
+and get involved by reaching out to the contacts listed.
+
+### Website documentation
+
+All components and patterns require usage, style, code, and accessibility
+guidance on the Carbon website. You can learn how to contribute documentation
+smoothly and utilize our markdown templates on the
+[documentation](contributing/documentation/) page. A list of components that
+still need documentation help can be found
+[here](https://github.com/carbon-design-system/carbon-website/issues/2085).
+
+### Fixing issues and bugs
+
+If you find a bug in the source code, you can help us by
+[filling out a bug report](https://github.com/carbon-design-system/carbon/issues/new/choose)
+and, ideally, submitting a pull request with a fix.
+
+### Icons and pictograms
+
+You can find instructions on how to create and contribute
+[UI icons](https://www.ibm.com/design/language/iconography/ui-icons/contribute),
```

Comment:
Should `App icons` and `Pictograms` also be capitalized in this case like they are on IDL or should they be lower case?

Reply:
I think they are only capitalized in the LHN but when I checked out the docs pages (i.e. design tab) they were lowercased so I will keep them as is.

Reply:
Got it!